packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -295,6 +295,7 @@ function FlowsheetDataGrid(props) {
         onCellSelect={handleCellSelection}
         onClearSelection={handleClearSelectedCells}
         onCellRangeSelect={handleCellRangeSelection}
+        hasColumnHeaders={false}
```

Comment:
The Flowsheet should have column headers by default.  The prop should be able to be set by consumers.

Reply:
Oh I misunderstood the requirements then. I'll change it.